<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color:white;
border-radius:6px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QFrame" name="header">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>130</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>Yu Gothic UI</family>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>FeedBack</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>5</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="min">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../resources.qrc">
                 <normaloff>:/icons/icons/minimize-sign.png</normaloff>:/icons/icons/minimize-sign.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="restor">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../resources.qrc">
                 <normaloff>:/icons/icons/maximize.png</normaloff>:/icons/icons/maximize.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="close">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../resources.qrc">
                 <normaloff>:/icons/icons/cross.png</normaloff>:/icons/icons/cross.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>90</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/icons/icons/rate.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="body_section">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QFrame" name="frame_7">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QCheckBox" name="star1">
               <property name="styleSheet">
                <string notr="true">QCheckBox::indicator {
    width:30px;
    height:30px; /* Corrected from 'hieght' to 'height' */
}

QCheckBox::indicator:unchecked { /* Corrected from 'indicater' to 'indicator' */
    image:url(&quot;:/icons/icons/star-unselect.png&quot;);
}


QCheckBox::indicator:checked{

	image:url(&quot;:/icons/icons/star.png&quot;);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="star2">
               <property name="styleSheet">
                <string notr="true">QCheckBox::indicator {
    width:30px;
    height:30px; /* Corrected from 'hieght' to 'height' */
}

QCheckBox::indicator:unchecked { /* Corrected from 'indicater' to 'indicator' */
    image:url(&quot;:/icons/icons/star-unselect.png&quot;);
}


QCheckBox::indicator:checked{

	image:url(&quot;:/icons/icons/star.png&quot;);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="star3">
               <property name="styleSheet">
                <string notr="true">QCheckBox::indicator {
    width:30px;
    height:30px; /* Corrected from 'hieght' to 'height' */
}

QCheckBox::indicator:unchecked { /* Corrected from 'indicater' to 'indicator' */
    image:url(&quot;:/icons/icons/star-unselect.png&quot;);
}


QCheckBox::indicator:checked{

	image:url(&quot;:/icons/icons/star.png&quot;);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="star4">
               <property name="styleSheet">
                <string notr="true">QCheckBox::indicator {
    width:30px;
    height:30px; /* Corrected from 'hieght' to 'height' */
}

QCheckBox::indicator:unchecked { /* Corrected from 'indicater' to 'indicator' */
    image:url(&quot;:/icons/icons/star-unselect.png&quot;);
}


QCheckBox::indicator:checked{

	image:url(&quot;:/icons/icons/star.png&quot;);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="star5">
               <property name="styleSheet">
                <string notr="true">QCheckBox::indicator {
    width:30px;
    height:30px; /* Corrected from 'hieght' to 'height' */
}

QCheckBox::indicator:unchecked { /* Corrected from 'indicater' to 'indicator' */
    image:url(&quot;:/icons/icons/star-unselect.png&quot;);
}


QCheckBox::indicator:checked{

	image:url(&quot;:/icons/icons/star.png&quot;);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	width:30px;
	height:25px;
	border-style:solid;
	border-color:gray;
	border-width:2px;
	margin:2px;
	padding:2px;
}</string>
            </property>
            <property name="placeholderText">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="comment">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-color: rgb(163, 163, 163);
border-style:solid;
border-width:2px;</string>
            </property>
            <property name="placeholderText">
             <string>Write comment</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="send">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">
QPushButton{
	background-color: qlineargradient(spread:pad, x1:0.205, y1:0.465955, x2:1, y2:1, stop:0.119318 rgba(95, 0, 160, 255), stop:0.744318 rgba(255, 0, 118, 255));
	color: rgb(255, 255, 255);	
	border-style:solid;
	border-color: rgb(199, 199, 199);
	border-width:1px;
}
QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(160, 16, 156, 255), stop:1 rgba(255, 90, 90, 255));
}
QPushButton:pressed {
	 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 12, 125, 255), stop:1 rgba(63, 223, 255, 255));
}</string>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QFrame" name="footer">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Version 1.0.0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>388</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
